ports/espressif/boards/lolin_c3_mini/mpconfigboard.mk

```diff
@@ -0,0 +1,12 @@
+USB_VID = 0x303A
+USB_PID = 0x80F5
+USB_PRODUCT = "Lolin C3 Mini"
+USB_MANUFACTURER = "Wemos"
```

Comment:
C3 does not use these as it does not have native USB.
For bluetooth, C3 boards require a creator/creation ID: https://github.com/creationid/

Reply:
Thanks @prplz !

For future me, when I get to this: Scott talks about Creation ID on the Deep Dive

https://github.com/adafruit/deep-dive-notes/search?q=creation+id

Reply:
Please remove these defines because they aren't used for C3.

Reply:
This is fixed in this commit:
https://github.com/adafruit/circuitpython/pull/6256/commits/47c5b3da1f4e8d196472bb0635ecbe2e72508eef